.stylelintrc

```diff
@@ -1,25 +1,19 @@
 {
-    "extends": [
-        "stylelint-config-recommended-scss",
-        "stylelint-config-css-modules"
-    ],
+    "extends": ["stylelint-config-recommended-scss", "stylelint-config-css-modules"],
     "syntax": "scss",
-    "plugins": [
-        "stylelint-order"
-    ],
+    "plugins": ["stylelint-order"],
     "rules": {
-        "declaration-no-important": true,
```

Comment:
Cannot this be ignored on the file level? then it might be easier to find and remove later. I think as a standard we should not use `important` unless really really needed.

Reply:
can't agree more, changed! :-)

Reply:
@eszthoff  important is removed completely 🎉 I didn't try the easiest option before 😅